octodns/provider/azuredns.py

```diff
@@ -121,6 +120,12 @@ def _params_for_SRV(self, data, key_name, azure_class):
                                       data['value']['target']))
         return {key_name: params}
 
+    def _params_for_TXT(self, data, key_name, azure_class):
+        if 'values' in data:
+            return {key_name: [azure_class([v]) for v in data['values']]}
+        else:  # API for TxtRecord has list of str, even for singleton
+            return {key_name: [azure_class([data['value']])]}
```

Comment:
I like your suggestion here. There is a similar structure on line 83 as well. Let me push a quick change then!